Gemfile.lock

```diff
@@ -26,7 +26,7 @@ GEM
     ffi (1.14.2)
     i18n (1.8.7)
       concurrent-ruby (~> 1.0)
-    json (2.5.1)
+    json (2.4.1)
```

Comment:
Why this change?

Reply:
no reason - reverted this! Thanks for the callout